BREAKING.md

```diff
@@ -2,6 +2,11 @@
 - [Property removed from IFluidDataStoreContext](#Property-removed-from-IFluidDataStoreContext)
 - [Changes to IFluidDataStoreFactory](#Changes-to-IFluidDataStoreFactory)
 - [FlushMode enum values renamed](#FlushMode-enum-values-renamed)
+- [name removed from ContainerSchema](#name-removed-from-ContainerSchema)
+- [Anonymous return types for container calls in client packages](#Anonymous-return-types-for-container-calls-in-client-packages)
+- [createContainer and getContainer response objects properties renamed](#createContainer-and-getContainer-response-objects-properties-renamed)
+- [tinylicious and azure clients createContainer now detached](#tinylicious-and-azure-clients-createContainer-now-detached)
+- [container id is returned from new attach() and not exposed on the container](#container-id-is-returned-from-new-attach()-and-not-exposed-on-the-container)
```

Comment:
nit: this link doesn't seem to work, probably the `()` needs to be removed?

Reply:
Hmm interesting. I actually checked this and it works in vscode. Removing them worked too so I did that but yea. Nice catch :)